.github/workflows/sycl-nightly.yml

```diff
@@ -80,6 +87,37 @@ jobs:
             image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
             target_devices: opencl:cpu
             tests_selector: cts
+
+          - name: E2E tests with dev igc on Intel Arc A-Series Graphics
+            runner: '["Linux", "arc"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:devigc
```

Comment:
Release drivers are more important than devigc, because that's what customers would use. Are we running full DG2 testing with release drivers after this change? \+ @sarnex , @YuriPlyakhin .